google-oauth-client/src/main/java/com/google/api/client/auth/oauth/OAuthHmacSha256Signer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010 Google Inc.
```

Comment:
Needs current copyright date, Inc. --> LLC 

and never, ever use (c) in anything, here or elsewhere. It is not recognized by the law, no matter how many places you find it.

Reply:
this has been updated